specifications/api/swagger.yaml

```diff
@@ -2202,13 +2202,55 @@ components:
         minimum_utxo_value:
           <<: *amount
           description: |
-            The minimum ada / Lovelace quantity required for new transaction outputs.
+            The absolute minimum quantity of ada required for a new transaction
+            output created with the wallet.
+
+            In general, the ledger rules require that every transaction output
+            has a minimum quantity of ada. This minimum quantity is determined
+            by an era-specific function whose value increases as the number of
+            different assets increases, as the quantity of any individual asset
+            increases, as the length of the target address increases, and if a
+            datum hash is added.
+
+            Therefore, the value reported by this field should only be viewed
+            as an absolute minimum, and only applies to outputs that send ada
+            (and no other assets) to Shelley-era addresses. If an output
+            contains other assets, specifies a datum hash, or sends funds to a
+            Byron-era address, then the minimum value required by the ledger
+            (and the wallet) will be higher than the value reported by this
+            field.
+
+            When using the wallet to construct or balance a transaction, if the
+            caller specifies an output with a non-zero ada quantity, then the
+            wallet will verify that the specified quantity is not less than the
+            minimum quantity required by the ledger, and if this verification
+            step fails, return an error that reports the required minimum. If
+            the caller specifies an output without an ada quantity, then the
+            wallet will automatically assign a minimal ada quantity to that
+            output.
+
+            In the Shelley, Allegra, and Mary eras, the `minimum_utxo_value`
+            field was equivalent to the ledger `minUTxOValue` protocol
+            parameter.
+
+            In the Alonzo era, the `minUTxOValue` protocol parameter was
+            replaced by the `coinsPerUTxOWord` protocol parameter. In this era,
+            the minimum ada quantity for an output was determined by
+            multiplying the `coinsPerUTxOWord` parameter by the length (in
+            8-byte words) of the in-memory representation of that output, which
+            was not dependent on the length of the address. Therefore, in this
```

Comment:
> of the in-memory representation of that output, which was not dependent on the length of the address.

Are you sure this is entirely correct? I thought it was an _upper bound_ of the in-memory representation.

Reply:
> > of the in-memory representation of that output, which was not dependent on the length of the address.
> 
> Are you sure this is entirely correct? 

This was my understanding, after previous discussions with the ledger team. 

> I thought it was an upper bound of the in-memory representation.

This would indeed explain why the address length does not affect the Alonzo-era minimum UTxO value computation.

I'll double-check with the ledger team and/or verify this statement by looking at the source code.

If it turns out that this description is inaccurate, I'll make another PR to fix it. 👍🏻

Reply:
@Anviking I've added a reminder to address this comment here: https://input-output.atlassian.net/browse/ADP-2039.